Ofqual.Recognition.Frontend.Web/appsettings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ApiAuthentication": {
+    "BaseUrl": ""
```

Comment:
Two things:
- Can we move the BaseUrl outside of ApiAuthentication as it's not actually related to the authentication?
- Could you update the README so that it contains definitions of what the appsettings are, so that people can derive what they should do in the future?